repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -260,10 +260,17 @@ def prepare_target_userspace(context, userspace_dir, enabled_repos, packages):
             _import_gpg_keys(context, install_root_dir, target_major_version)
 
         api.current_logger().debug('Installing cloudlinux-release')
-        context.call(['rpm', '--import', 'https://repo.cloudlinux.com/cloudlinux/security/RPM-GPG-KEY-CloudLinux'], callback_raw=utils.logging_handler)
-        context.call(['dnf', '-y', 'localinstall', 'https://repo.cloudlinux.com/cloudlinux/migrate/release-files/cloudlinux/8/x86_64/cloudlinux8-release-current.x86_64.rpm'], callback_raw=utils.logging_handler)
-
-        enable_spacewalk_module(context)
+        context.call(['rpm', '--import', 'https://repo.cloudlinux.com/cloudlinux/security/RPM-GPG-KEY-CloudLinux'],
+                     callback_raw=utils.logging_handler)
+        cloudlinux_release_url = 'https://repo.cloudlinux.com/cloudlinux/migrate/release-files' \
+                                 '/cloudlinux/{version}/x86_64/cloudlinux{version}-release-current.x86_64.rpm' \
+                                 ''.format(version=target_major_version)
```

Comment:
Minor codestyle fix suggestion: brackets are preferred to backslashes to break the long lines [according to PEP8](https://peps.python.org/pep-0008/#maximum-line-length).
```suggestion
        cloudlinux_release_url = (
            'https://repo.cloudlinux.com/cloudlinux/migrate/release-files'
            '/cloudlinux/{version}/x86_64/cloudlinux{version}-release-current.x86_64.rpm'
        ).format(version=target_major_version)
```